src/components/20-molecules/datepicker/index.js

```diff
@@ -143,6 +145,11 @@ class AXADatepicker extends NoShadowDOM {
   }
 
   static get properties() {
+    const now = new Date();
+    // formally mark the (internal) default date, s.t. it can be distinguished
+    // from dates that are set by applications (exploiting the malleability
+    // of system objects...)
+    now.type = 'default';
```

Comment:
Isn't that dangerous if the standard class Date adds its own prop `type` in future? Maybe name it like `axaType`?

Reply:
Browser system classes like `Date` are long-term stable, any change must go through a lengthy spec process. We can wait for that to happen...